packages/administration/assets/src/styles/form.scss

```diff
@@ -0,0 +1,12 @@
+.form-check {
+    margin: 0.25rem;
+}
+
+.form-check-input:hover {
+    border-color: var(--border-color-hovered);
+    cursor: pointer;
+}
+
+.form-check-label:hover {
+    cursor: pointer;
```

Comment:
hover on label on a disabled element should not initiate hover state, or? 

![Snímek obrazovky 2025-11-05 v 16 36 55](https://github.com/user-attachments/assets/8bba1648-3eea-4ad8-9726-e16acf444b88)